### PR TITLE
add access log forwarding to stdout 

### PR DESCRIPTION
**Pull request by dmfine**
So that it will be recorded on the docker host and forwarded to papertrail.

### CODE REVIEW

**Comment by dmfine** [comment]
The previous PR had the access_log directive at the top level, which it turns out is not allowed. Nginx starts when it's under the server block. QA has some issue at the moment which is preventing me from testing it as thoroughly as I'd like.

**Comment by seanh** [comment]
The commit was based off an old, broken version of the master branch. I've rebased it for you

File: conf/nginx.conf [hunk 1]
```diff
@@ -33,7 +33,7 @@ http {
         include /var/lib/hypothesis/nginx_envsubst.conf;
         listen 9083;
         merge_slashes off;
-
+        access_log /dev/stdout;
```
Comment (seanh) [inline conf/nginx.conf]: What's the difference between putting this in the `server { ... }` block, vs putting it higher up in the `http { ... }` block? It looks like putting it in the `server` block might mean that it applies only to that one virtual server, when the `nginx.conf` might contain multiple `server`'s in the future.

Comment (seanh) [reply conf/nginx.conf]: Yeah, this should be in the `http { ... }`. See https://www.nginx.com/resources/wiki/start/topics/examples/full/ and https://nginx.org/en/docs/http/ngx_http_log_module.html#access_log. I've pushed a commit to fix it

**Review by seanh — APPROVED** [review]

File: conf/nginx.conf (hunk 1 above)
Comment (dmfine) [reply conf/nginx.conf]: future proofing +1

**Pull request merged**